src/mmw/js/src/modeling/models.js

```diff
@@ -130,12 +164,32 @@ var ScenarioModel = Backbone.Model.extend({
         name: '',
         is_current_conditions: false,
         modifications: null,      // ModificationsCollection
-        active: false
+        active: false,
+        uid: null
```

Comment:
I was actually going to initially comment that I'd prefer `id` just like every other resource we have.  So now there's _three_ options.  Not sure it's necessary to have special behavior for the user, I'd prefer to have things consistent.  `user.user_id` and `user.uid` both seem a little redundant, also.  When saved on another object, I think `userId` makes the most sense and is consistent with how we would store `projectId/scenarioId` (or rather, snake_cased).